appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: 1.0.{build}
+
+branches:
+  only:
+    - master
+
+image: Visual Studio 2017
+
+# Pull down all submodule code
+init:
+  - git config --global core.autocrlf input
+
+# Cache the third party code as it can take 20+ min to build
+cache:
+  - third_party
+
+platform: x64
```

Comment:
Should the value of platform and configuration be `%CONFIGURATION%` and `%PLATFORM%`?

Reply:
No this is defining what %CONFIGURATION% and %PLATFORM% will be.  We can set multiple values but I think the build would take to long with the coreclr.